tests/test_converters.py

```diff
@@ -0,0 +1,568 @@
+# This File is for testing atmos indicators that fit neither in test_precip or test_temperature
+# Expected values might be the same as for the indices tests, see test_indices comments.
+from __future__ import annotations
```

Comment:
are these all existing tests that were ported here?

Reply:
Precisely. It just so happens that 90% of the tests in `test_indicators.py` were testing conversion tools.